src/components/PermissionsManagement/PermissionListItem.jsx

```diff
@@ -147,13 +147,17 @@ function PermissionListItem(props) {
               className="icon-button"
               color={getColor()}
               onClick={() => {
+                // eslint-disable-next-line no-debugger
                 // const state = howManySubpermsInRole !== 'None';
                 setSubpermissions(subperms, howManySubpermsInRole !== 'All');
                 // eslint-disable-next-line react/destructuring-assignment
                 props.onChange();
               }}
               // eslint-disable-next-line react/destructuring-assignment
-              disabled={!props.hasPermission('putRole')}
+              disabled={
+                !(props.hasPermission('putRole') || props.hasPermission('postRole')) ||
+                immutablePermissions.includes(permission)
+              }
```

Comment:
```suggestion
              disabled={immutablePermissions.includes(permission)}
```
Wouldn't work for individual permissions.